portality/bll/services/todo.py

```diff
@@ -61,8 +61,59 @@ def group_stats(self, group_id):
                 elif b["key"] == constants.APPLICATION_TYPE_UPDATE_REQUEST:
                     stats["by_status"][bucket["key"]]["update_requests"] = b["doc_count"]
 
+        stats["historical_numbers"] = self.historical_numbers(eg)
+
+        return stats
+
+    def historical_numbers(self, editor_group:models.EditorGroup):
```

Comment:
There are 2 functions: historical_numbers and historical_count which do different things with different arguments, and the naming is a bit awkward.  Maybe we can align the count/number thing and give the functions more specific names, like:

group_finished_provenance_counts

user_finished_provenance_counts

I would then put the `year` at least as an argument, and default it to the current year, like

```python
def group_finished_provenance_counts(editor_group, year=None):
    if year is None:
        year = dates.now_str(dates.FMT_YEAR)
```